packages/mdc-textfield/foundation.js

```diff
@@ -98,12 +127,24 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.adapter_.removeHelptextAttr(ARIA_HIDDEN);
   }
 
+  transitionEnd_(evt) {
+    const {BOTTOM_LINE_ACTIVE} = MDCTextfieldFoundation.cssClasses;
+
+    // We need to wait for the bottom line to entirely transparent
+    // before removing the class. If we do not, we see the line start to
+    // scale down before disappearing - sheehana
```

Comment:
Is it normal for us to put specific author names in comments? Not sure I've seen us do it before in this codebase.

Reply:
I don't know if we've every had a rule one way or the other. I can take it out.